examples/api-routes-apollo-server-and-client/lib/init-apollo.js

```diff
@@ -0,0 +1,56 @@
+import ApolloClient from 'apollo-client'
+import { SchemaLink } from 'apollo-link-schema'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { HttpLink } from 'apollo-link-http'
+import { schema } from '../apollo/schema'
+import gql from 'graphql-tag'
+
+let apolloClient = null
+
+function create (initialState) {
+  // Check out https://github.com/zeit/next.js/pull/4611 if you want to use the AWSAppSyncClient
+  const isBrowser = typeof window !== 'undefined'
```

Comment:
I am not sure if code elimination works if we assign `typeof window !== 'undefined'` to a variable. 🤔